saleor/graphql/core/types/common.py

```diff
@@ -121,6 +122,10 @@ class MenuError(Error):
     code = MenuErrorCode(description="The error code.", required=True)
 
 
+class OrderSettingsError(Error):
```

Comment:
You should add `OrderSettingsError` to `SALEOR_ERROR_CODE_ENUMS` in `saleor/saleor/graphql/core/utils/error_codes.py`